### PR TITLE
Fix typos in the about section

### PR DESCRIPTION
**Pull request by Luthaf**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -148,17 +148,16 @@ <h5 class="modal-title">Chemiscope: interactive structure/property explorer for
...
-                            Chemiscope is free and open source, distributed under the BSD license itself, and the GPL license for the Jmol dependency.
-                            It is developed by <a href="https://guillaume.fraux.fr">Guillaume Fraux</a> in the
-                            <a href="https://cosmo.epfl.ch">COSMO lab</a> at EPFL, with support from the
-                            <a href="http://nccr-marvel.ch">NCCR MARVEL</a> and <a href="http://max-centre.eu">MAX</a> european center of excellence.
-                            Early development was influenced by the <a href="https://github.com/lab-cosmo/isv"> Interactive Sketch-map Visualizer</a>
+                            Chemiscope is free and open source, distributed under the BSD license. It is developed by
+                            <a href="https://guillaume.fraux.fr">Guillaume Fraux</a> in the <a href="https://cosmo.epfl.ch">COSMO lab</a> at EPFL,
+                            with support from the <a href="http://nccr-marvel.ch">NCCR MARVEL</a> and <a href="http://max-centre.eu">MAX</a> european
+                            center of excellence. Early development was influenced by the
```
Comment (cbenmahm) [inline app/index.html:159]: I would use the capitalisation used in the max website : "European Centre of Excellence "

File: app/index.html [hunk 2]
```diff
@@ -148,17 +148,16 @@ <h5 class="modal-title">Chemiscope: interactive structure/property explorer for
                     </div>
                     <div class="modal-body">
                         <p>
-                            Chemiscope is a tool for interactive exploration of databases of materials and molecular, correlating local and global
-                            structure descriptors with the properties of the systems. Structural properties are represented by a descriptor mapped
-                            onto a smaller space using a dimensionality reduction algorithm.
+                            Chemiscope is a tool for interactive exploration of databases of materials and molecules, correlating local and global
+                            structural representation with the properties of the systems.
```
Comment (cbenmahm) [inline app/index.html:152]: representations*

**Review by cbenmahm — COMMENTED** [review]
looks good to me, just a couple of cosmetic changes

File: app/index.html (hunk 2 above)
Comment (Luthaf) [reply app/index.html:152]: Instead of `structural representation`?

Comment (cbenmahm) [reply app/index.html:152]: I meant `structural representations`

**Pull request merged**